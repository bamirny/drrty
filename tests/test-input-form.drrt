def int main(){
	str head;

	head = "This is the header for my form!";
	makeHeader(head);

	makeText("Please enter your name in here");
	makeInput("");/* Takes in class name*/

	createElement("button","Submit","");

	return 0;

}