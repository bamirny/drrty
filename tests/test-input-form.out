<h1>This is the header for my form!</h1>
<p>Please enter your name in here</p>
<input class="">
<button class="">Submit</button>
